### DISCUSSION TITLE
Zoom plot of an output file?

### DISCUSSION THREAD

**Discussion by AndresSepulveda** [OpenDrift/opendrift#821] [Q&A] [answered]
Hi,

How do I do a plot of a portion of the domain using an output file?

I have this already:

import opendrift
import numpy as np

o = opendrift.open("archivo.nc", elements=np.arange(0,392000,100))

o.plot(filename="archivo.png",linecolor="z",fast = True)
o.animation(filename="archivo.mp4",linecolor="z",fast = True)

Cheers,

 Andrés

**Comment by knutfrode** [comment, marked as answer]
You can add `corners=[lonmin, lonmax, latmin, latmax]`

like in this example:
https://opendrift.github.io/gallery/example_grid.html
